nh_clinical/tests/common/nh_clinical_test_utils.py

```diff
@@ -166,3 +198,32 @@ def copy_instance_variable_if_exists(self, caller, variable_name):
         instance_variable_value = getattr(self, variable_name, None)
         if instance_variable_value:
             setattr(caller, variable_name, instance_variable_value)
+
+    # TODO EOBS-753: Move to patient model? Is just a generally useful method.
```

Comment:
Sorry put the comment in a bad place, I'm referring to the `get_open_activities_for_patient` method.